task-sdk/src/airflow/sdk/log.py

```diff
@@ -140,7 +140,7 @@ def emit(self, record: logging.LogRecord):
 
 
 @cache
-def logging_processors(enable_pretty_log: bool, mask_secrets: bool = True):
+def logging_processors(enable_pretty_log: bool, mask_secrets: bool = True, enable_colors: bool = True):
```

Comment:
Can we just use the same name as the config? 
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#colored-console-log